sonarqube-postgresql-template.yaml

```diff
@@ -137,24 +137,6 @@ objects:
     stringData:
       password: ${POSTGRESQL_PASSWORD}
 
-  - apiVersion: v1
-    kind: ImageStream
-    metadata:
-      labels:
-        app: sonarqube
-        role: ${SONARQUBE_ROLE}
-      name: sonarqube
-    spec:
-      tags:
-        - annotations:
-            description: The SonarQube Docker image
-            tags: sonarqube
-          from:
-            kind: DockerImage
-            name: "${SONARQUBE_SOURCE_IMAGE_NAME}:${SONARQUBE_VERSION}"
-          importPolicy: {}
-          name: "${SONARQUBE_VERSION}"
-
```

Comment:
The ImageStream section is here because the `oc new-build` step is only performed if you are building a custom or updated image.  The default is to use the published image.  Therefore this section should be retained.

Reply:
Sounds good, lets keep that section in there